Source/WebCore/platform/graphics/WidthIterator.cpp

```diff
@@ -637,13 +714,13 @@ void WidthIterator::advance(unsigned offset, GlyphBuffer& glyphBuffer)
     float startingRunWidth = m_runWidthSoFar;
 
     if (m_run.is8Bit()) {
-        Latin1TextIterator textIterator(m_run.data8(m_currentCharacterIndex), m_currentCharacterIndex, offset, length);
+        Latin1TextIterator textIterator(m_run.data8(0), m_currentCharacterIndex, offset, length);
```

Comment:
Well this doesn't seem right.... WidthIterator's API is supposed to allow the caller to say 
```
myWidthIterator.advance(0);
myWidthIterator.advance(1);
myWidthIterator.advance(2);
```